iree/compiler/Dialect/VM/IR/VMBase.td

```diff
@@ -418,6 +469,11 @@ def VM_AnyType : AnyTypeOf<[
   VM_AnyRef,
 ]>;
 
+def VM_PrimitiveType : AnyTypeOf<[
+  AnyIntOfWidths<[8, 16, 32, 64]>,
```

Comment:
Is 64 forward looking? I don't think VM generally supports this now?

Reply:
Correct; the idea here is that we could store 64-bit elements in a list but when read back they'd be converted to int32 (today) and that would allow easier interop with app code that may be wrapping lists of int64s. Unless that just makes things more difficult for type conversion... hmm. I'll remove for now so that we aren't confused later :)